### DISCUSSION TITLE
Deep Learning for UAV Infrastructure Inspection (Project 187)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#21] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #187](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Deep%20Learning%20for%20UAV%20Infrastructure%20Inspection)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by anani-moss** [comment]
Hello World !

**Reply by robertogl** [reply]
Hello Jiyon. Great to see you are going to be working on this project! Do not hesitate to ask questions or write comments about the project in this discussion.

**Comment by karthickai** [comment]
Hi Innovators! I'm Karthick master's student specializing in computer vision. I'm looking forward to working together. Here is my linkedin profile: https://www.linkedin.com/in/karthickhps/ . Thanks, Mathworks, for this great initiative!

**Reply by robertogl** [reply]
Hello Karthick,

Sorry for the late response. It is great to have you onboard! We are really glad that you like our initiative, it is important for us to know how we are doing and any feedback is always much appreciated!

Please do not hesitate to ask any questions related to the project.
Good luck!

**Comment by Yuvi744** [comment]
Hi All

**Comment by Tulutas** [comment]
Hello,
I'm Tolga Ulutas from Yıldız Technical University. I am a PhD student in the control and automation department. I am also looking forward to working together.

**Comment by huntermanana** [comment]
hello,
how do i get started to work on this?

**Reply by robertogl** [reply]
Hi @huntermanana,

Great to know you are interested in working on this project. My suggestion would be to get familiar with the UAV Toolbox and Unreal engine. In particular, how to simulate flight scenarios and sensors in Unreal and how to use them with the UAV Toolbox.
The project description has links to examples and useful resources. 

Please let me know if you need anything else.
